azure-pipelines.yml

```
@@ -0,0 +1,83 @@
+pool:
+  name: Hosted VS2017

```

As microsoft updates build agents, build failures can happen as there can be different versions of SDKs installed. This is a notorious problem in the multi-platform development world and isn't as important for a project like DNN. My best practice I've been using is specifying `vmImage` instead of `name`. This would give us more granular control over what Hosted-Agent is being used to reduce the risk of random build failures.

```suggestion
  vmImage: windows-2019
```

This may not work as a drop-in replacement. We may need to add a `jobs` root node and specify `pool` on each `job`. See the docs below for more details
- https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops&tabs=yaml